titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -512,6 +518,27 @@ public static Task toCoreTask(com.netflix.titus.grpc.protogen.Task grpcTask) {
         return builder.build();
     }
 
+    static Map<String, String> buildAttributesMapForCoreTask(com.netflix.titus.grpc.protogen.Task grpcTask) {
+        Map<String, String> attributes = new HashMap<>(grpcTask.getAttributesMap());
+        if (grpcTask.hasLogLocation()) {
+            LogLocation logLocation = grpcTask.getLogLocation();
+            if (logLocation != null) {
```

Comment:
Redundant check as we already call `grpcTask.hasLogLocation()` which checks for that.